src/App.tsx

```diff
@@ -1,14 +1,46 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoadedTodos, setIsLoadedTodos] = useState(false);
+  const [checkedTodo, setCheckedTodo] = useState<Todo | null>(null);
+  const [inputState, setInputState] = useState('');
+  const [toggleFilter, setToggleFilter] = useState<boolean | null>(null);
```

Comment:
you can set here 'all' select option by default. Also, name toggle doesn`t suitable in that case, because here is not a toggle but options in select. So good name will be 'option, setOption' :)

Reply:
totally agree,
let's use an enum for that
```suggestion
  const [filter, setFilter] = useState<FilterType>(FilterType.None);
```